pipelines/skylab/multiome/Multiome.wdl

```diff
@@ -34,8 +34,18 @@ workflow Multiome {
         Array[File] atac_r1_fastq
         Array[File] atac_r2_fastq
         Array[File] atac_r3_fastq
-        # BWA input
+        
+        # BWA tar reference
         File tar_bwa_reference
+        # BWA machine type
```

Comment:
Let's double check with the team that it's ok to put these inputs at the input section of the workflow definition.